juju/python-libjuju
# [JUJU-1733] Revisit unitrun example

#### Description

This revisits the example located at `examples/unitrun.py` and reflects the change happened to the `unit.run()`. The most crucial thing is the added line : `await action.run()`.

#### QA Steps

At the top directory, run:

```
python examples/unitrun.py
```

Aside from the asyncio dump (will be fixed in the near future), we should see:

```
Action status: completed
Action results: {'return-code': 0, 'stdout': '10.42.51.209\n'}
```
